pandas/tests/io/pytables/test_read.py

```diff
@@ -0,0 +1,270 @@
+import pytest
+from warnings import catch_warnings
+
+import numpy as np
```

Comment:
there are many more selection / query type tests that are in test_common. move here